### ISSUE TITLE
assets: unmantained dependency react-alert

### ISSUE THREAD

**Issue by dennybiasiolli** [apache/incubator-superset#2700]
`react-alert` component's code is unmantained and it uses deprecated code that will be removed in React version 16. ([link](https://github.com/schiehll/react-alert))

It returns the following warnings during `npm test`:
```
Warning: Accessing PropTypes via the main React package is deprecated. Use the prop-types package from npm instead.
Warning: AsyncCreatableSelect: React.createClass is deprecated and will be removed in version 16. Use plain JavaScript classes instead. If you're not yet ready to migrate, create-react-class is available on npm as a drop-in replacement.
```

What do you think about fork & update the component in order to avoid deprecation errors when updating to React 16?

**Comment by mistercrunch** [comment]
Either forking it or using some equivalent component that is maintained.

**Comment by dennybiasiolli** [comment]
@mistercrunch fork & update in progress ([link here](https://github.com/dennybiasiolli/react-alert/tree/updating_deps)).
I'm also trying to contact the author in order to unlock the "no maintenance intended" status

**Comment by mistercrunch** [comment]
For you to be able to maintain the package you'll need access to push releases to `npm`, hopefully the author can grant you that. The main repo is where the work (PRs, issues, ...) should be done, so either you move his repo to your GH org or he grants you admin rights.

**Comment by dennybiasiolli** [comment]
Yes thanks, I'm waiting for a response from @schiehll, the author of the component, in order to have admin rights on the repo or at least on `npm` package

**Comment by dennybiasiolli** [comment]
Update: the author is working on a v2 of `react-alert` component, with updated code and dependency. The new version should be compatible with the latest react version without warnings.
As soon as it's released I can test the new version and eventually open a new PR updating the component dependency.

**Comment by mistercrunch** [comment]
Awesome. Grateful that you are taking this on!

**Comment by dennybiasiolli** [comment]
Closed by #2923
